src/molecules/ChipLarge.tsx

```diff
@@ -12,39 +22,68 @@ const styles = StyleSheet.create({
     borderRadius: 11,
     height: 22,
   } as ViewStyle,
-  IconAndText: {
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
   Icon: {
     marginRight: 4,
   },
   Text: registeredTextStyle('titleBold', colors.ulisse, 'chip-text'),
   TextDark: registeredTextStyle('titleBold', colors.uma, 'chip-text-dark'),
 });
 
+type containerStyle = {
+  paddingLeft?: FlexStyle['paddingLeft'];
+  paddingRight?: FlexStyle['paddingRight'];
+  paddingTop?: FlexStyle['paddingTop'];
+  paddingBottom?: FlexStyle['paddingBottom'];
+  marginLeft?: FlexStyle['marginLeft'];
+  marginRight?: FlexStyle['marginRight'];
+  marginTop?: FlexStyle['marginTop'];
+  marginBottom?: FlexStyle['marginBottom'];
```

Comment:
this is also something that shouldn't be exposed to users of the design system lib. I guess we have a few possible combinations of these values, and that choice is what must be exposed instead. From a quick overview of the twin PR in `urbi-react-native`, it looks like it's either only `layoutStyles.margins`, or a combo of `layoutStyles.margins` + `paddingLeft: 0, paddingTop: 0`, and `marginLeft: 0`; these must correspond to something on our DS in Figma, so they must be associated with a name that makes sense within that context.

So, if we have more than 2 possible combinations of these parameters,`containerStyle` should become a union type of strings (something like `'noPaddings' | 'topLeftMargins' | 'whatever'`). If not, it could even simply become an optional boolean prop that's only set in the non-default case (like, e.g., `noPadding: boolean`). The _values_ associated with those options should be stored here, inside `styles`.

Reply:
Thanks. Commits https://github.com/urbi-mobility/urbi-react-native/commit/f12589cd0164f21c1c4f1f2b1338dec7d6308036 10b9a3f